modules/snap_community_charts/charts.php

```diff
@@ -93,29 +100,22 @@
     <li>a shift in precipitation from snow to rain, which impacts water storage capacity and surface water availability</li>
 </ul>
 
+<p>Note: Precipitation may occur as either rain or snow, but is reported for all months in terms of rainwater equivalent.</p>
+
 <p>Warmer, drier spring weather may also be an indicator for increased fire risk. In many locations, winter temperatures are projected to increase dramatically. Warmer winters may favor growth of species that are less cold-hardy (including desirable crops and invasive species), or it may decrease snowpack and increase the frequency of rain-on-snow events that impact wildlife. Higher temperatures across all seasons will likely impact permafrost and land-fast ice. <a id="derive_help">Learn more about how we derived the community climate outlooks</a>
 </p>
 
-
 <div class="about_dialog" id="modal_block" style="display: none;">
 
-<p><strong>Data sources: </strong>historical <abbr title="Parameter-elevation Regressions on Independent Slopes Model">PRISM</abbr> climatology data and downscaled outputs averaged from five <abbr title="General Circulation Models">GCMs</abbr>. <a href="/methods/downscaling">Learn more about how we downscale climate data from global to regional scales.</a></p>
+<p><strong>Data sources: </strong>Historical <abbr title="Parameter-elevation Regressions on Independent Slopes Model">PRISM</abbr> and CRU TS 3.2 climatology data (1961-1990) and downscaled outputs averaged from five <abbr title="General Circulation Models">GCMs</abbr>. <a href="https://www.snap.uaf.edu/methods/downscaling">Learn more about how we downscale climate data from global to regional scales.</a></p>
 
 <p><strong>We averaged results to smooth out short-term variability. </strong>Results are averaged across decades to lessen the influence of normal year-to-year climate variability on projected values. Averaging also tends to make overall projection trends clearer. Uncertainty is associated with each of these graphed values, and stems from:</p>
 
 <ul>
     <li>modeling of atmospheric and oceanic movements used to create <abbr title="General Circulation Models">GCMs</abbr></li>
-    <li>the <abbr title="Parameter-elevation Regressions on Independent Slopes Model">PRISM</abbr> downscaling process&gt;</li>
+    <li>the downscaling process&gt;</li>
```

Comment:
Good question? That's why it's nice to have someone else reviewing the code :smiley: